docs/commands/json.clear.md

```diff
@@ -18,7 +18,7 @@ is JSONPath to specify. Default is root `$`. Nonexisting paths are ignored.
 
 ## Return
 
-JSON.CLEAR returns an integer reply specified as the number of values cleared. 
+JSON.CLEAR returns an integer reply specified as the number of matching JSON non-zero numerical values cleared.
```

Comment:
Maybe a bit simpler to grasp such as:
`the number of matching JSON values which are numerical and non-zero that were cleared.`